exporter/clickhouseexporter/config.go

```diff
@@ -145,6 +172,48 @@ func (cfg *Config) shouldCreateSchema() bool {
 	return cfg.CreateSchema
 }
 
+func (cfg *Config) ValidateMetricTableNames() error {
+	if len(cfg.MetricsTableName) == 0 {
```

Comment:
Based on the function name, I would assume this would validate the table names struct.

This if statement seems to return when the old value is empty. The old value would always be empty if they're using the new struct. Is this validating correctly? Based on the unit tests it looks like this simply returns a warning if both fields are used.

Reply:
to simplify the logic I completely removed this function, will add a note to readme about the suggestion to use the newly introduced field